### ISSUE TITLE
Is the computation of the sentence level accuracy performed using teacher forcing?

### ISSUE THREAD

**Issue by shawntan** [yli1/CGPS#1]
I'm not familiar with Tensorflow, but it seems from the code that the `l` variable here (https://github.com/yli1/CGPS/blob/master/simple_model.py#L199) is used both for computing the loss and for sentence accuracy. Does this mean that the ground-truth sequence is used during decoding for evaluation?

**Comment by yli1** [comment]
Thank you for asking. l is logit for model output, and it is compared with ground-truth y to compute loss and sentence accuracy. The ground-truth y is not used during decoding from input x to model output, but y is used to compare with model output and compute evaluation metric (sentence accuracy). So the proposed method does not use teacher forcing to compute sentence level accuracy.

**Comment by shawntan** [comment]
I see, thanks!
